src/main/java/duke/Duke.java

```diff
@@ -45,7 +45,7 @@ private static void respondToInput() {
         while (!userInput.equals("bye")) {
             try {
                 if (userInput.equals("list")) {
-                    displayList();
+                    displayList(tasks);
                 } else if (checkIsInputEquals(userInput, "done ")) {
```

Comment:
Can just use userInput.substring(0, 4).contains("done") or something like that, because the current method is very long and it's a bit confusing.